packages/react-charts/src/components/Patterns/examples/patterns.md

```diff
@@ -317,12 +317,10 @@ class InteractivePieLegendChart extends React.Component {
     // Returns legend data styled per hiddenSeries
     this.getLegendData = () => {
       const { hiddenSeries } = this.state;
-      return this.series.map((s, index) => {
-        return {
+      return this.series.map((s, index) => ({
           ...s.legendItem, // name property
           ...getInteractiveLegendItemStyles(hiddenSeries.has(index)) // hidden styles
-        };
-      });
+        }));
```

Comment:
Did Eslint add the extra parens? I understand parens are required to wrap a component, but doesn't seem necessary for JSON.

Reply:
Yeah, it added them because the signature when from:

```js
() => {
  return {}
}
```

To:

```js
() => ({ })
```

The parentheses here are required, as otherwise the JS engine cannot distinguish the body of a function from an object-literal.